### PR TITLE
feat(kit): `InputPhoneInternational` can be customized with `Icon` / `Tooltip`

### PR DESCRIPTION
**Pull request by nsbarsukov**

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/kit/components/input-phone-international/input-phone-international.template.html [hunk 1]
```diff
@@ -37,6 +37,8 @@
         (ngModelChange)="onValueChange($event)"
     />
 
+    <ng-content select="tui-icon" />
```
Comment (splincode) [inline projects/kit/components/input-phone-international/input-phone-international.template.html:40]: `<ng-content select="tui-icon, img" />` Can I put my image source?

**Review by waterplea — APPROVED** [review]

File: projects/kit/components/input-phone-international/input-phone-international.template.html (hunk 1 above)
Comment (nsbarsukov) [reply projects/kit/components/input-phone-international/input-phone-international.template.html:40]: @waterplea I had the same thoughts too.
Probably, I should add `TODO` (for `5.0`)

File: projects/demo/src/modules/components/input-phone-international/examples/4/index.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+@Component({
+    standalone: true,
+    imports: [FormsModule, TuiInputPhoneInternational, TuiIcon, TuiTooltip],
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    encapsulation,
+    changeDetection,
+    providers: [
+        tuiInputPhoneInternationalOptionsProvider({
```
[the remark this answers is not in the document]
Comment (nsbarsukov) [reply projects/demo/src/modules/components/input-phone-international/examples/4/index.ts:22]: It is required for correct work in Stackblitz
See:
* https://github.com/taiga-family/taiga-ui/pull/8603

(I notice that we should add providers for the second example too)

File: projects/kit/components/input-phone-international/input-phone-international.template.html (hunk 1 above)
Comment (nsbarsukov) [reply projects/kit/components/input-phone-international/input-phone-international.template.html]: @splincode fixed in https://github.com/taiga-family/taiga-ui/pull/8750/commits/a4d1e45baa2ae3bd562964e57a618701694caf85

@waterplea fixed in https://github.com/taiga-family/taiga-ui/pull/8750/commits/baece221754662f6bd9793fdf3e68921fc299e59

**Pull request merged**